### DISCUSSION TITLE
🇩🇪 The Mask of the Sun

### DISCUSSION THREAD

**Discussion by kironia** [eltikia/eltikia.github.io#5] [Q&A] [answered]
Was bedeute die Videobeschreibungen?

**Comment by Stagyrite** [comment, marked as answer]
Zuerst übersetze ich der Titel.

> Das Geheimnis der Aztekenmaske.

Dann ich spreche wie eine Hexe. Es ist vielleicht wie die Maske benutzen.

> Was ist schön dabei?

Fast niemand kennt diese Hexe, aber das ist mir egal.
